tests/integration/features.go

```diff
@@ -766,11 +766,11 @@ func CheckTracesInstall(builder *features.FeatureBuilder) *features.FeatureBuild
 				waitDuration,
 				tickDuration,
 				stepfuncs.WithNameF(
-					stepfuncs.ReleaseFormatter("%s-sumologic-traces-gateway"),
+					stepfuncs.ReleaseFormatter("%s-sumologic-traces-sampler"),
```

Comment:
Why not `gateway`?

Reply:
It's a bug :) Few lines below there's check for gateway, and there's no check for sampler :)